### DISCUSSION TITLE
New contributor interested in GSoC 2026

### DISCUSSION THREAD

**Discussion by SanketA01** [MDAnalysis/mdanalysis#5137] [GSoC Discussions]
Hi everyone,
​My name is Sanket, and I am a third-year Information Technology student. I'm very interested in MDAnalysis and your work in analyzing molecular dynamics simulations.
​I am a beginner to open source, but I have experience with Python and data science libraries like Pandas and NumPy. My goal is to learn how to contribute and eventually apply for Google Summer ofCode 2026 with your organization.
​I've been looking at the GitHub repository, and I'd like to start by helping with a small task. Could anyone please point me toward some "good first issues" or "beginner-friendly" bugs?
​Thanks for your help!
​Best,
Sanket

**Comment by IAlibay** [comment]
@SanketA01 welcome to MDAnalysis. For beginner friendly issues, please see our issue tracker for issues that are labelled as "help wanted", "difficulty easy", or "gsoc starter". If possible, I would suggest tackling one or two of the latter two labels, and then focus on a larger / more difficult issue. 

Regarding GSoC, please note the following: 
1) MDAnalysis has not made a decision on whether or not they will apply to GSoC next year (in fact, I don't even know that Google has announced GSoC for 2026). Even after making a decision to apply, we won't know if we're going to be accepted as an organisation until ~ February - April 2026. There is always a chance that we won't be able to participate in the coming year.
2) Generally (but not always) MDAnalysis GSoC projects tend to require some previous knowledge of molecular simulations.

**Comment by iGufrankhan** [comment]
Hi everyone,

My name is Gufran Khan, and I’m a third-year Information Technology student. I’m very interested in MDAnalysis and your work in analyzing molecular dynamics simulations.
I’m a beginner to open source, but I have experience with Python, machine learning, and popular libraries such as NumPy, Pandas, TensorFlow, Keras, and Scikit-learn.
I’d love to start contributing to MDAnalysis, learn about open-source development, and improve my skills by helping with real-world projects.
I’ve been exploring the GitHub repository and documentation, and I’d like to begin with a beginner-friendly task. Could someone please guide me toward some “good first issues” or easy bugs to get started?

Thank you for your time and help!

Best regards,
Gufran Khan

**Comment by UtkarshSahay123** [comment]
Hi I am new to this open-source world and currently I am in second year of my college and I want to contribute in gsoc in 2026 so can anyone please guide me

**Comment by IAlibay** [comment]
All, please see my comment here for everything you need to know re: contributing and GSoC when it comes to MDAnalysis: https://github.com/MDAnalysis/mdanalysis/discussions/5137#discussioncomment-14797637

We will not be providing individualised guidance.

**Comment by aseem102** [comment]
Hii everyone
i am Aseem Nakade passionate to work as data analyst and to work on machine learning models 
currently i am pursuing undergraduate course and like to contribute in organisation please guide me how to start 
i am new contributor intrested in gsoc 2026

**Comment by IAlibay** [comment]
Please see here for FAQs regarding GSoC 2026: https://github.com/MDAnalysis/mdanalysis/discussions/5204
